commands/expire.md

```diff
@@ -31,6 +31,18 @@ will be `del`, not `expired`).
 [del]: /commands/del
 [ntf]: /topics/notifications
 
+## Options
+
+The `EXPIRE` command supports a set of options since Redis 7.0:
+
+* `NX` -- Set expiry only when the key has no expiry
+* `XX` -- Set expiry only when the key has an existing expiry
+* `GT` -- Set expiry only when the new expiry is greater than current one
+* `LT` -- Set expiry only when the new expiry is less than current one
+
+A non-volatile key is treated as an infinite ttl.
```

Comment:
maybe TTL should be upper case? (being an acronym)

Reply:
and maybe it would be clearer if we add "for the purpose of GT / LT"?
i.e.
```
A non-volatile key is treated as an infinite TTL for the purpose of GT and LT.
```